features/environment.py

```diff
@@ -436,16 +399,11 @@ def _should_skip_tags(context: Context, tags: List) -> str:
                         return ""
 
                     cloud_manager = context.config.cloud_manager
-                    if cloud_manager and cloud_manager.missing_env_vars():
-                        return "".join(
-                            (
-                                "Skipped: {} machine_type requires:\n".format(
-                                    machine_type
-                                ),
-                                *cloud_manager.format_missing_env_vars(
-                                    cloud_manager.missing_env_vars()
-                                ),
-                            )
+                    cloud_creds_path = context.config.cloud_credentials_path
+                    if cloud_manager and not cloud_creds_path:
+                        return "Skipped: {} machine_type requires:\n{}".format(
+                            machine_type,
+                            "UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH",
```

Comment:
pycloudlib checks a couple places for the pycloudlib.toml file so this envvar isn't totally necessary (see https://github.com/canonical/pycloudlib/blob/main/pycloudlib/config.py#L36-L42).

So I think this should only print a warning instead of preventing the test from running

Reply:
That is true, but the reason I am adding this variable is because we might need a different set of credentials to the ones we currently have on the machine. But I agree that if we don't need that, we can just drop it

Reply:
(Mentioned this earlier but putting it here for the public record): I like the environment variable as an override, I just think that it shouldn't be required. Right now the test won't run if the envvar isn't set. Ideally I could choose between:
1. set the envvar to a specific toml file
2. don't set the envvar so that pycloudlib finds my normal toml file

Reply:
for what it's worth: cloud-init integration tests work by not requiring you to specify any cloud-specific environment files and just relies on a properly configured pycloudlib.toml or PYCLOUDLIB_CONFIG environment variable as override. I don't know if there is strong need to establish a new environment variable up in UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH,  which could ultimately be understood anyway by pycloudlib if UA jenkins jobs/Jenkinsfile provided PYCLOUDLIB_CONFIG environment variable.